src/test/addrman_tests.cpp

```diff
@@ -79,15 +79,10 @@ static CDataStream AddrmanToStream(const AddrManSerializationMock& _addrman)
 
 class AddrManTest : public AddrMan
 {
-private:
-    bool deterministic;
 public:
-    explicit AddrManTest(bool makeDeterministic = true,
-                         std::vector<bool> asmap = std::vector<bool>())
-        : AddrMan(asmap, makeDeterministic, /* consistency_check_ratio */ 100)
-    {
-        deterministic = makeDeterministic;
-    }
+    explicit AddrManTest(std::vector<bool> asmap = std::vector<bool>())
+        : AddrMan(asmap, /*deterministic=*/true, /* consistency_check_ratio */ 100)
```

Comment:
nit, feel free to ignore:
```suggestion
        : AddrMan(asmap, /*deterministic=*/ true, /*consistency_check_ratio=*/ 100)
```

Reply:
oops. I'll be more consistent with these comments in future.

Reply:
I think `/*deterministic=*/true` is the "clang-format-correct" formatting.

Reply:
Thanks Marco. That was my understanding too. I should have also formatted `consistency_check_ratio` in that way.